webpack.config.js

```diff
@@ -2,10 +2,13 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
-  entry: './src/index.js',
+  entry: './src/drag.js',
+  devServer: {
```

Comment:
- [ ] Please use index.js as your starting point, and import drag.js and complete.js in there